docs/reference/ingest/enrich.asciidoc

```diff
@@ -144,6 +144,13 @@ to create an ingest pipeline.
 Include an <<enrich-processor,enrich processor>>
 that uses your enrich policy.
 
+When defining an enrich processor,
+you must include the following options:
+
+* *Field*, the same match field defined in your enrich policy.
```

Comment:
`field` is not entirely accurate:

`field` configures what field the enrich processor will retrieved the value from the document being ingested, which is then used to query against the match field in the enrich index. (the match field in the enrich index is the same as the `match_field` configured in the policy, actually it used to create the enrich index)

Not sure how this can be condensed into a readable sentence.

Reply:
Thanks! That's helpful. I'll work on a rewording.

Reply:
Added 1ed9793a26990d31a1f22c8ada4aedd6aef4a013 to clarify this.

Here's the revised bullet:

```
*  The *field* used to match incoming documents
    to documents in the enrich index.
+
This field should be included in incoming documents.
To match, this field must contain the exact
value of the match field of a document in the enrich index.

Also made a few other non-substantial changes. Thanks again!